README.md

```diff
@@ -277,14 +277,21 @@ For example, use gdal to convert each downloaded file from NetCDF (`*.nc`) to Ti
      process: !shell
         command: 'gdal_translate -a_srs "+proj=latlong +datum=WGS84" {parent_dir}/{filename} {parent_dir}/{file_stem}.tif'
         expect_file: '{parent_dir}/{file_stem}.tif'
+        input_files:('^(?P<base>.*hdf)', ['{base}', '{base}.xml'])
```

Comment:
A tuple didn't work for me.  I had to change the config to be a list

Reply:
I had to change the value for the _input_files_ to:

input_files: ['^(?P<brdf_base>.*BRDF)/(?P<collection>.*)/(?P<ymd>[0-9]{4}\.[0-9]{2}\.[0-9]{2})/(?P<basename>.*)(?P<hdf_ext>.hdf)(?P<xml_ext>.xml)?', ['{brdf_base}/{collection}/{ymd}/{basename}{hdf_ext}', '{brdf_base}/{collection}/{ymd}/{basename}{hdf_ext}.xml']]

Which while length, gave the flexibility needed for enabling the .hdf and the .XML to handled whichever file was provided.

The expected file example I used was:

expect_file: '/tmp/conversion/BRDF/{collection}/{ymd}/{basename}.h5'

Reply:
Might be worth updating the README for the BRDF example where the _input_files_ is explicitly used.